compiler/luci/service/src/Nodes/CircleSum.cpp

```diff
@@ -14,15 +14,11 @@
  * limitations under the License.
  */
 
-#include <luci/Service/CircleShapeSignatureInference.h>
-
 namespace luci
 {
 
-ShapeSignature ssinf::Algorithm::visit(const luci::CircleSum *node)
-{
-  return legalized_signature(
-    reduced_signature(node->input(), node->reduction_indices(), node->keep_dims()));
-}
+// Shape Inference Algorithm will be added
```

Comment:
(optional) we may remove all similar files like this now and add them later when needed.

Reply:
I will apply it :)